modules/applications/pages/ui-builder/download-and-launch.adoc

```diff
@@ -30,22 +30,21 @@ include::example$ui-builder/development/docker-compose.yml[]
 
 INFO: You can override and configure all xref:ui-builder-docker-installation.adoc#environment-variables[environment variables] in the `docker-compose.yml` file.
 
-- Create a `nginx` configuration file `nginx.conf` with the content below:
+== Configure the bonita-ui-proxy image
```

Comment:
I think we should move this section to https://bonitasoft-bonita-doc-deploy-pr-2809.surge.sh/bonita/latest/applications/ui-builder/ui-builder-docker-installation 

And make changes to https://bonitasoft-bonita-doc-deploy-pr-2809.surge.sh/bonita/latest/applications/ui-builder/ui-builder-docker-installation so that we have 2 sections : one for configuring bonita-ui-builder and one for configuring bonita-ui-proxy

Reply:
Done